milli/Cargo.toml

```diff
@@ -29,8 +29,8 @@ geoutils = "0.5.1"
 grenad = { version = "0.4.5", default-features = false, features = [
     "rayon", "tempfile"
 ] }
-heed = { git = "https://github.com/meilisearch/heed", tag = "v0.12.7", default-features = false, features = [
-    "lmdb", "read-txn-no-tls"
+heed = { version = "0.20.0-alpha.9", default-features = false, features = [
+    "serde-json", "serde-bincode", "read-txn-no-tls"
```

Comment:
Check features

Reply:
It's okay because we don't need the (default) "serde" feature to serialize some types.